plugins/@grouparoo/onesignal/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,25 @@
+import { DestinationMappingOptionsMethod } from "@grouparoo/core";
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod = async () => {
+  return {
+    labels: {
+      property: {
+        singular: "OneSignal Tag",
+        plural: "OneSignal Tags",
+      },
+      group: {
+        singular: "OneSignal Group Tag",
+        plural: "OneSignal Group Tags",
+      },
+    },
+    properties: {
+      required: [{ key: "external_user_id", type: "string" }],
+      known: [
+        { key: "first_name", type: "string", important: true },
```

Comment:
Are these known tags?

Reply:
These tags have a special meaning in OneSignal: https://documentation.onesignal.com/docs/add-user-data-tags#user-names--preferences

![image](https://user-images.githubusercontent.com/4368928/110733890-24db6880-81fd-11eb-8c71-c104acfd42c4.png)